keyboards/kprepublic/bm40hsrgb/rev2/config.h

```diff
@@ -0,0 +1,142 @@
+  /*==== use exp() and sin() ====*/
+  #define RGBLIGHT_EFFECT_BREATHE_CENTER 1.85  // 1 to 2.7
+  #define RGBLIGHT_EFFECT_BREATHE_MAX    255   // 0 to 255
+
+
+#    define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+#    define RGB_MATRIX_KEYPRESSES
+
+#    define RGB_DISABLE_WHEN_USB_SUSPENDED // turn off effects when suspended
+#    define RGB_MATRIX_MAXIMUM_BRIGHTNESS 180 // Limit to vendor-recommended value
+// RGB Matrix Animation modes. Explicitly enabled
+// For full list of effects, see:
+// https://docs.qmk.fm/#/feature_rgb_matrix?id=rgb-matrix-effects
+
+//some effects disabled arbitarily to save space
+
+#    define ENABLE_RGB_MATRIX_ALPHAS_MODS
+#    define ENABLE_RGB_MATRIX_GRADIENT_UP_DOWN
+#    define ENABLE_RGB_MATRIX_GRADIENT_LEFT_RIGHT
+#    define ENABLE_RGB_MATRIX_BREATHING
+#    define ENABLE_RGB_MATRIX_BAND_SAT
+#    define ENABLE_RGB_MATRIX_BAND_VAL
+#    define ENABLE_RGB_MATRIX_BAND_PINWHEEL_SAT
+#    define ENABLE_RGB_MATRIX_BAND_PINWHEEL_VAL
+#    define ENABLE_RGB_MATRIX_BAND_SPIRAL_SAT
+#    define ENABLE_RGB_MATRIX_BAND_SPIRAL_VAL
+#    define ENABLE_RGB_MATRIX_CYCLE_ALL
+#    define ENABLE_RGB_MATRIX_CYCLE_LEFT_RIGHT
+#    define ENABLE_RGB_MATRIX_CYCLE_UP_DOWN
+// #    define ENABLE_RGB_MATRIX_RAINBOW_MOVING_CHEVRON
+// #    define ENABLE_RGB_MATRIX_CYCLE_OUT_IN
+// #    define ENABLE_RGB_MATRIX_CYCLE_OUT_IN_DUAL
+// #    define ENABLE_RGB_MATRIX_CYCLE_PINWHEEL
+// #    define ENABLE_RGB_MATRIX_CYCLE_SPIRAL
+// #    define ENABLE_RGB_MATRIX_DUAL_BEACON
+// #    define ENABLE_RGB_MATRIX_RAINBOW_BEACON
+// #    define ENABLE_RGB_MATRIX_RAINBOW_PINWHEELS
+// #    define ENABLE_RGB_MATRIX_RAINDROPS
+// #    define ENABLE_RGB_MATRIX_JELLYBEAN_RAINDROPS
+// #    define ENABLE_RGB_MATRIX_HUE_BREATHING
+// #    define ENABLE_RGB_MATRIX_HUE_PENDULUM
+// #    define ENABLE_RGB_MATRIX_HUE_WAVE
+// #    define ENABLE_RGB_MATRIX_PIXEL_RAIN
+// #    define ENABLE_RGB_MATRIX_PIXEL_FLOW
+// #    define ENABLE_RGB_MATRIX_PIXEL_FRACTAL
+// enabled only if RGB_MATRIX_FRAMEBUFFER_EFFECTS is defined
+#    define ENABLE_RGB_MATRIX_TYPING_HEATMAP
+#    define ENABLE_RGB_MATRIX_DIGITAL_RAIN
+// enabled only of RGB_MATRIX_KEYPRESSES or RGB_MATRIX_KEYRELEASES is defined
+#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE
+#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE
+//#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_WIDE
+#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTIWIDE
+//#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_CROSS
+#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTICROSS
+//#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_NEXUS
+#    define ENABLE_RGB_MATRIX_SOLID_REACTIVE_MULTINEXUS
+//#    define ENABLE_RGB_MATRIX_SPLASH
+#    define ENABLE_RGB_MATRIX_MULTISPLASH
+//#    define ENABLE_RGB_MATRIX_SOLID_SPLASH
+#    define ENABLE_RGB_MATRIX_SOLID_MULTISPLASH
+
+
+#    define DRIVER_ADDR_1 0b1010000
+#    define DRIVER_COUNT 2
```

Comment:
I think this is one difference between #16689 and this (other than the hybrid issi+ws2812 RGB matrix driver)

`DRIVER_COUNT` should be 1, since there's only one ISSI driver.